libraries/AC_PID/AC_P_1D.cpp

```diff
@@ -0,0 +1,171 @@
+/// @file	AC_P_1D.cpp
+/// @brief	Generic P algorithm
+
+#include <AP_Math/AP_Math.h>
+#include "AC_P_1D.h"
+
+const AP_Param::GroupInfo AC_P_1D::var_info[] = {
+    // @Param: P
+    // @DisplayName: P Proportional Gain
+    // @Description: P Gain which produces an output value that is proportional to the current error value
+    AP_GROUPINFO("P",    0, AC_P_1D, _kp, 0),
```

Comment:
don't use ID 0

Reply:
moved to id 1, txs

Reply:
I think what happened is that Tridge asked me to move it (as seen above), I intended to move it but then realised that we can't because AC_P_1D replaces AC_P and moving this would mean a bunch of parameter conversions.

I could add a comment into the code but we're never going to look at that comment again after this review is done so let's just try and remember.